### PR TITLE
Backport upstream patch to fix embedded icons

### PR DESCRIPTION
**Pull request by wjt**
https://gitlab.gnome.org/GNOME/simple-scan/-/issues/390
https://gitlab.gnome.org/GNOME/simple-scan/-/merge_requests/269

Merged upstream but not yet part of any release.

### CODE REVIEW

File: org.gnome.SimpleScan.json [hunk 1]
```diff
@@ -250,6 +250,13 @@
                         "type": "git",
                         "tag-pattern": "^([\\d.]+)$"
                     }
+                },
+                {
+                    "type": "patch",
+                    "paths": [
+                        "simple-scan-Use-RDNN-app-ID.patch"
+                    ],
+                    "use-git": true
```
Comment (hfiguiere) [inline org.gnome.SimpleScan.json:259]:
```suggestion
                    "path": "simple-scan-Use-RDNN-app-ID.patch"
```

Comment (wjt) [reply org.gnome.SimpleScan.json:259]: Isn't that functionally equivalent?

Comment (hfiguiere) [reply org.gnome.SimpleScan.json]: less lines is better. if you don't need it don't put it

Comment (wjt) [reply org.gnome.SimpleScan.json]: I don't agree but it's not worth arguing about! I pushed a new version.

**Pull request merged**